artilleryTests/preprod/preprod-rosetta-stability-test.yml

```diff
@@ -0,0 +1,173 @@
+      name: "High Load ~5000 req/min"
+
+  plugins:
+    # keep or remove as needed
+    ensure: {}
+    expect:
+      expectDefault200: true
+      outputFormat: silent
+      reportFailuresAsErrors: true
+    metrics-by-endpoint:
+      useOnlyRequestNames: true
+      metricsNamespace: "latency_metrics"
+
+# --------------------------------------------------------------------------
+# SCENARIOS
+# --------------------------------------------------------------------------
+scenarios:
+  - name: "Network status test"
+    flow:
+      - post:
+          url: "/network/status"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "metadata": {}
+            }
+
+  - name: "Account balance test"
+    flow:
+      - post:
+          url: "/account/balance"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "account_identifier": {
+                "address": "{{ address }}"
+              }
+            }
+
+  - name: "Account coins test"
+    flow:
+      - post:
+          url: "/account/coins"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "account_identifier": {
+                "address": "{{ address }}"
+              },
+              "include_mempool": true
+            }
+
+  - name: "Block test"
+    flow:
+      - post:
+          url: "/block"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "block_identifier": {
+                "index": {{ block_index }},
+                "hash": "{{ block_hash }}"
+              }
+            }
+
+  - name: "Block transaction test"
+    flow:
+      - post:
+          url: "/block/transaction"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "block_identifier": {
+                "index": {{ block_index }},
+                "hash": "{{ block_hash }}"
+              },
+              "transaction_identifier": {
+                "hash": "{{ transaction_hash }}"
+              }
+            }
+
+  - name: "Search transactions test"
+    flow:
+      - post:
+          url: "/search/transactions"
+          body: |
+            {
+              "network_identifier": {
+                "blockchain": "cardano",
+                "network": "preprod"
+              },
+              "transaction_identifier": {
+                "hash": "{{ transaction_hash }}"
+              }
+            }
+
+  - name: "Construction metadata test"
```

Comment:
we don't actually need those but you can keep them... just clutters a file a bit